openfe/utils/visualization.py

```diff
@@ -115,20 +118,26 @@ def _draw_molecules(
     highlight_color: Tuple[float, float, float, float]
         RGBA tuple for the default highlight color used in the mapping
         visualization
+    atom_mapping: Dict[Tuple[int,int], Dict[int, int]], optional
+        used to align the molecules to each othter for clearer visualization,
+        default None
```

Comment:
A little more detail would be useful in the docstring here -- namely, how, do I interpret this object? We're mapping a tuple to a dict; how do I understand what each int corresponds to?